www/main/navigator/utils/load-shabad.js

```diff
@@ -1,5 +1,10 @@
 import banidb from '../../banidb';
 
 export const loadShabad = (shabadID, lineID) => {
-  return banidb.loadShabad(shabadID, lineID).then(rows => rows);
+  return banidb
+    .loadShabad(shabadID, lineID)
+    .then(rows => rows)
+    .catch(() => {
+      console.log('err while loading shabad, please contact dev team');
```

Comment:
@inderpreetsingh Do we need to show this to end user?

Reply:
we can do that in a separate PR, let's merge this fix so that its not a blocker in QA.